code/_onclick/cyborg.dm

```diff
@@ -182,5 +182,6 @@
 	A.attack_robot(src)
 
 /atom/proc/attack_robot(mob/user)
-	attack_ai(user)
-	return
```

Comment:
Okay so coincidentally this breaks a couple things. We can't keep attack_ai() here because parent procs (/obj/item) are going to also attempt to call attack_ai(). Looks like I have to refactor these two procs ughhh